qutip/solver/result.py

```diff
@@ -894,78 +919,208 @@ def __repr__(self):
 
     def __add__(self, other):
         if not isinstance(other, MultiTrajResult):
-            return NotImplemented
+            raise NotImplementedError("Can only add two multi traj results")
```

Comment:
Why not let python raise the error when no appropriate functions is found?
Not that I think this would happen, it stop other project to develop something compatible with our results.

Reply:
Sorry, this was a mistake, reverted the change